test/node/get-server-address-ipc.js

```diff
@@ -0,0 +1,39 @@
+'use strict';
```

Comment:
@belochub btw, after some thinking about it I'm now on your side about the fate of these tests if/when we'll be returning subclasses of Node.js core servers. Not that I'll be determined to remove them and I still think that extra tests aren't a bad thing even if there's not much need in them, but both keeping and removing these tests will sound fine to me.